python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -784,7 +784,8 @@ def test_dataframe_to_string():
     assert got.split() == expect.split()
 
 
-def test_dataframe_to_string_wide():
+def test_dataframe_to_string_wide(monkeypatch):
+    monkeypatch.setenv("COLUMNS", 79)
```

Comment:
Why do we need to set this?

Reply:
Resolving this as it is necessary to mock the terminal width.

Reply:
Connected offline - but it looks like this test will currently fail on terminals that are longer than 82 characters wide.  e.g.,

```
$ COLUMNS=100 pytest cudf/tests/test_dataframe.py::test_dataframe_to_string_wide
```

Thus the monkeypatch